docs/externalGrading.md

```diff
@@ -238,36 +238,58 @@ Running PrairieLearn locally with externally graded question support looks somet
 - Create an empty directory to use to share job data between containers. 
     - This can live anywhere, but needs to be created first and referenced in 
       the docker launch command.
+    - This command is copy-pastable for Windows PowerShell, MacOS, and Linux.
 ```bash
-mkdir ${HOME}/pl_ag_jobs
+mkdir "$HOME/pl_ag_jobs"
 ```
 
-- Modify your PL docker run call to look something like:
+- Modify your PL docker run call to include the jobs directory.
+
+On MacOS and Linux, `cd` to your course directory and copy-paste the following command:
 ```sh
 docker run -it --rm -p 3000:3000 \
-    -v ${PWD}:/course `# Map your current directly in as course content` \
-    -v ${HOME}/pl_ag_jobs:/jobs `# Map jobs directory into /jobs` \
-    -e HOST_JOBS_DIR=$HOME/pl_ag_jobs \
+    -v "$PWD":/course `# Map your current directory in as course content` \
+    -v "$HOME/pl_ag_jobs:/jobs" `# Map jobs directory into /jobs` \
+    -e HOST_JOBS_DIR="$HOME/pl_ag_jobs" \
     -v /var/run/docker.sock:/var/run/docker.sock `# Mount docker into itself so container can spawn others` \
     prairielearn/prairielearn 
 ```
 
-**Note:** The sequence above is verified to work on Windows 10 as well as MacOS and Linux docker. 
+On Windows PowerShell, `cd` to your course directory and copy the following command **except** with your own username in `HOST_JOBS_DIR`:
+```sh
+docker run -it --rm -p 3000:3000 -v $PWD\:/course -v $HOME\pl_ag_jobs:/jobs -e HOST_JOBS_DIR=/c/Users/Tim/pl_ag_jobs -v /var/run/docker.sock:/var/run/docker.sock prairielearn/prairielearn 
+```
+
+**Note** the following about `HOST_JOBS_DIR` on PowerShell:
+
+* Use Unix-style paths (i.e., use `/c/Users/Tim/pl_ag_jobs`, **not** `C:\Users\Tim\pl_ag_jobs`)
+* Use the full path rather than `$HOME` (i.e., use `/c/Users/Tim/pl_ag_jobs`, **not** `$HOME/pl_ag_jobs`).
 
-##### Common issues
+##### Windows errors and quirks
 
-Windows users might see problems running grading jobs. The most common issue is the:
+Windows users might see problems running grading jobs:
 
 ```bash
 standard_init_linux.go:207: exec user process caused "no such file or directory"
 ```
 
-The error occurs during grading as a result of an OS new-line incompatibility with the `entrypoint` script in the externally
+This error occurs during grading as a result of an OS new-line incompatibility with the `entrypoint` script in the externally
 graded question. One solution for this is to make a `.gitattributes` files in your PL repository with the line
 `*.sh text eol=lf`. This tells the GitHub client to write the script files in native Linux
 format instead of converting them for Windows (which breaks mapping them back into docker). 
 This mimics the [`.gitattributes` file in the main PrairieLearn repo](https://github.com/PrairieLearn/PrairieLearn/blob/master/.gitattributes).
 
+
```

Comment:
Could we add a separate sub-heading to separate the error above ("exec user process") from the error below ("invalid mode: /grade")? I don't have a good idea what it should look like, but at the moment I worry that the error below is not clearly separated from the error above.

Reply:
I added 6th-level headings. Not sure if too deep.